cloudinit/apport.py

```diff
@@ -110,30 +116,57 @@ def attach_cloud_info(report, ui=None):
                 report["CloudName"] = "None"
 
 
+def attach_installer_files(report, ui=None):
```

Comment:
@dbungert in the future we might lifyt this type of function into `/usr/share/apport/package-hooks/subiquity.py`  or maybe surfacing an APPORT_FILES module-level attribute which could be a dict keyed on "<path>" with <apport_label> as the value. In either approach subiquity could control what files is cares to attach to a report. We can talk about this approach for next cycle if it sounds compelling.